abiquo-firstboot.py

```diff
@@ -597,5 +683,9 @@ def __init__(self):
     try:
         signal.signal(signal.SIGINT, signal_handler)   
         ret = mainWindow()
+
+        plymouth_mode = commands.getoutput("/usr/sbin/plymouth-set-default-theme")
+        if plymouth_mode == "details":
+            commands.getstatus("/usr/sbin/plymouth-set-default-theme text --rebuild-initrd")
```

Comment:
This plymouth stuff is not needed as the init script kills plymouth the first thing.